examples/pypi-source-deps/pixi.toml

```diff
@@ -0,0 +1,29 @@
+[project]
+name = "pypi-source-deps"
+version = "0.1.0"
+description = "Add a short description here"
+authors = ["Tim de Jager <tim@prefix.dev>"]
+channels = ["conda-forge"]
+platforms = ["osx-arm64", "linux-64", "win-64"]
+
+[tasks]
+hello = "python -c 'from minimal_project import module;module.hello()'"
```

Comment:
You could use `importlib.metadata.version("module")` to check if the package is installed or if it just lies conveniently on the python path

See https://github.com/Quantco/polarify/blob/d39cd4a1ff852fb1e60a368fed27c25901d1b8ef/polarify/__init__.py#L10

Reply:
Okay added that check to `__init__.py` that's what you meant right :)?

Reply:
Yes 👍🏻 
But in that case (or maybe only in internal tests) even raise an error since uv should always install it properly and something went wrong when it didn't work.

Reply:
Ok will change that :)